diff --git a/pycbc/filter/simd_correlate_ccode.cpp b/pycbc/filter/simd_correlate_ccode.cpp
index ab19fe638..1ad8e2cb4 100644
--- a/pycbc/filter/simd_correlate_ccode.cpp
+++ b/pycbc/filter/simd_correlate_ccode.cpp
@@ -33,7 +33,10 @@
 #include <stdlib.h>
 #include <complex>
 #include <omp.h>
-#include <x86intrin.h>
+
+#if (defined(__x86_64__) || defined(__i386__))
+    #include <x86intrin.h>
+#endif
 
 #ifdef __AVX2__
 #define _HAVE_AVX2 1
