diff --git a/setup.py b/setup.py
index 4003c7556..6885b1b81 100755
--- a/setup.py
+++ b/setup.py
@@ -44,7 +44,6 @@ install_requires =  setup_requires + ['Mako>=1.0.1',
                       'pillow',
                       'h5py>=2.10',
                       'jinja2',
-                      'lalsuite!=7.2',
                       'astropy>=2.0.3,!=4.2.1,!=4.0.5',
                       'mpld3>=0.3',
                       'lscsoft-glue>=1.59.3',
