diff --git a/pycbc/events/simd_threshold_ccode.cpp b/pycbc/events/simd_threshold_ccode.cpp
index 95a0554964..ef3ac6307d 100644
--- a/pycbc/events/simd_threshold_ccode.cpp
+++ b/pycbc/events/simd_threshold_ccode.cpp
@@ -48,9 +48,7 @@
  * compartmentalize SIMD code from OpenMP code.
  */

-#ifndef __clang__
-    #include <omp.h>
-#endif
+#include <omp.h>
 #include <stdint.h> // For uint32_t, int64_t
 #include <complex>
 #include <math.h> // For M_SQRT2
diff --git a/pycbc/filter/simd_correlate_ccode.cpp b/pycbc/filter/simd_correlate_ccode.cpp
index 5746cb02ae..ab19fe638f 100644
--- a/pycbc/filter/simd_correlate_ccode.cpp
+++ b/pycbc/filter/simd_correlate_ccode.cpp
@@ -32,9 +32,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <complex>
-#ifndef __clang__
-    #include <omp.h>
-#endif
+#include <omp.h>
 #include <x86intrin.h>

 #ifdef __AVX2__
